analytics/src/main/java/com/segment/analytics/Analytics.java

```diff
@@ -287,6 +300,7 @@ public Analytics build() {
           .setClient(client)
           .setRequestInterceptor(new RequestInterceptor() {
             @Override public void intercept(RequestFacade request) {
+              request.addHeader(USER_AGENT_HEADER, userAgent);
```

Comment:
I realized there's a potential for a bug here.

```
Builder builder = new Builder().userAgent("foo");

Analytics first = builder.build();

Analytics second = builder.userAgent("bar").build();
```

both first and second will use "bar" as the user agent. Not a big deal and we can fix this later, but just something to note.

Reply:
meh, was easy to fix. added now.